app/src/components/SoldArtworkList.tsx

```diff
@@ -37,25 +37,22 @@ class SoldArtworkList extends React.Component<ContractProps, SoldArtworkListStat
     const currentAccount = this.props.accounts[0];
     let eventsFound = 0;
     const soldInfoArray: any[] = [];
+    const filter = {filter: {from: currentAccount}}
+    const options = {filter, fromBlock: 0}
```

Comment:
Potential issue with this: most likely will be way too far back to look on Rinkeby, but may not be because we know we did not deploy any contracts back then: is there a way to determine what the first blocknumber is when we deploy new contracts? If so should replace this with that.